redwood-yoga/build.gradle

```diff
@@ -19,3 +19,14 @@ kotlin {
     }
   }
 }
+
+
+spotless {
+  kotlin {
+    targetExclude(
+      // Internal MIT-licensed files from Meta.
+      fileTree("src/commonMain/kotlin/app/cash/redwood/yoga/internal").filter { it.isFile() },
+      fileTree("src/commonTest/kotlin/app/cash/redwood/yoga/internal").filter { it.isFile() },
```

Comment:
No wildcard support? It's unclear from Spotless docs.

Reply:
There doesn't appear to be unfortunately! `"src/**/kotlin/app/cash/redwood/yoga/internal"` is invalid.